cmd/podmanV2/containers/attach.go

```diff
@@ -0,0 +1,62 @@
+package containers
+
+import (
+	"os"
+
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	attachDescription = "The podman attach command allows you to attach to a running container using the container's ID or name, either to view its ongoing output or to control it interactively."
+	attachCommand     = &cobra.Command{
+		Use:   "attach [flags] CONTAINER",
+		Short: "Attach to a running container",
+		Long:  attachDescription,
+		RunE:  attach,
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 1 || (len(args) == 0 && !cmd.Flag("latest").Changed) {
+				return errors.Errorf("attach requires the name or id of one running container or the latest flag")
+			}
+			return nil
+		},
+		PersistentPreRunE: preRunE,
+		Example: `podman attach ctrID
+  podman attach 1234
+  podman attach --no-stdin foobar`,
+	}
+)
+
+var (
+	attachOpts entities.AttachOptions
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: attachCommand,
+	})
+	attachCommand.SetHelpTemplate(registry.HelpTemplate())
+	attachCommand.SetUsageTemplate(registry.UsageTemplate())
+	flags := attachCommand.Flags()
+	// TODO uncomment after create is merged
+	//flags.StringVar(&attachCommand.DetachKeys, "detach-keys", common.getDefaultDetachKeys(), "Select the key sequence for detaching a container. Format is a single character `[a-Z]` or a comma separated sequence of `ctrl-<value>`, where `<value>` is one of: `a-z`, `@`, `^`, `[`, `\\`, `]`, `^` or `_`")
```

Comment:
This isn't create specific, it sets it for this attach session only

Reply:
i know what im saying is there is that the common function it calls there wont exist until the create pr lands.

Reply:
resolved.